features/steps/flaw.py

```diff
@@ -1,7 +1,6 @@
 from behave import given, when, then
 from selenium.webdriver.common.by import By
 
-from features.steps.common_steps import *
```

Comment:
After removing this line, is there any impact on login in this scenario?

Reply:
No. Because common_steps.py in the dir steps, cucumber can find it automatically. I test it, works fine.